ipl/main.c

```diff
@@ -2431,8 +2451,8 @@ ment_t ment_tools[] = {
 	MDEF_CAPTION("-------- Misc --------", 0xFF0AB9E6),
 	MDEF_HANDLER("Dump package1", dump_package1),
 	MDEF_HANDLER("Fix battery de-sync", fix_battery_desync),
-	MDEF_HANDLER("Remove archive bit (switch folder)", fix_sd_switch_attr),
-	//MDEF_HANDLER("Remove archive bit (all sd files)", fix_sd_all_attr),
+	MDEF_HANDLER("Fix /switch archive bit", fix_sd_switch_attr),
+	MDEF_HANDLER("Fix ALL archive bit", fix_sd_switch_attr),
```

Comment:
Hmm, `fix` does not quite fit here (it fits with attributes).

What about `Unset \'switch\' archive attributes` and `Unset all archive attributes`
(They are big again but less than before. And don't use case for "all", so it can be similar with every menu option)

Additionally, you forgot to use `fix_sd_all_attr`. Both of them unset the switch folder and its contents.

Reply:
> Additionally, you forgot to use fix_sd_all_attr. Both of them unset the switch folder and its contents.

Yep, another bug I found quickly after pushing this up.